opennms-config-model/src/main/resources/xsds/jmx-config.xsd

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
as discussed: let's restrict just to the same types as we have in the Jaxb class.